examples/plot_mdm_spd_helper.py

```diff
@@ -0,0 +1,145 @@
+import numpy
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+
+plt.style.use('seaborn')
+
+fontsize=15
+matplotlib.rc('font', size=fontsize)
+matplotlib.rc('text', usetex=True)
+matplotlib.rc('legend', fontsize=fontsize)
+matplotlib.rc('font',
+              family='times',
+              serif=['Computer Modern Roman'],
+              monospace=['Computer Modern Typewriter'])
+
+EPS=1e-8
+
+class DatasetSPD_2D():
+
+    def __init__(self,n_samples,n_features,n_classes):
+        self.n_samples=n_samples
+        self.n_features=n_features
+        self.n_classes=n_classes
+        self.data_helper=DataHelper()
+
+    def generate_sample_dataset(self):
+        X,Y=self.easy_gauss()
+        X,Y=self.data_helper.shuffle(X,Y)
+        return X,Y
+
+    def mat2vec(self,mat):
+        n=mat.shape[-1]
+        return mat[numpy.tril_indices(n)]
+
+    # returns one random rotation of dimension n
+    def random_rotation(self):
```

Comment:
Could you use `random_uniform` from the class `SpecialOrhogonal`?
```
so = SpecialOrthogonal(n=3)
rot = so.random_uniform()
```

BTW: the random_uniform method has a pb, see issue #407.

Reply:
I switched to using it.